app/main.py

```diff
@@ -1,2 +1,65 @@
+from typing import Hashable, Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity: int = 8
+        self.size: int = 0
+        self.load_factor: float = 0.66
+        self.hash_table: list = [None] * self.capacity
+
+    def __len__(self) -> int:
+        print(self.capacity)
+        return self.size
+
+    def __setitem__(self, key: Hashable, value: Any) -> None:
+        threshhold = int(self.capacity * self.load_factor)
+        if self.capacity - threshhold > self.hash_table.count(None):
+            self.resize()
+
+        index = self.get_index_from_hash(key)
+        while True:
```

Comment:
Try to avoid so many conditions here
Consider to add some condition in while instead of using just while True, as you did in __getitem__ method